src/desktop/package.json

```diff
@@ -19,43 +18,48 @@
     "url": "git+https://github.com/iotaledger/trinity-wallet.git"
   },
   "scripts": {
-    "start": "concurrently --kill-others \"npm run devserver\" \"cross-env NODE_ENV=development electron -r @babel/register main.js\"",
+    "start": "concurrently --kill-others \"npm run devserver\" \"cross-env NODE_ENV=development electron -r @babel/register native/Index.js\"",
     "postinstall": "electron-builder install-app-deps",
-    "devserver": "node server.js",
-    "style:shots": "node scripts/puppeteer.js",
+    "devserver": "node scripts/webpackServer.js",
     "build": "rimraf dist && npm run build-main && npm run build-app",
     "build-main": "./node_modules/.bin/cross-env NODE_ENV=production webpack --env=production --config webpack.config/config.main.js",
     "build-app": "./node_modules/.bin/cross-env NODE_ENV=production webpack --env=production --config webpack.config/config.app.js",
     "compile": "npm run build && npm run compile:linux && npm run compile:mac && npm run compile:win",
     "compile:linux": "./node_modules/.bin/electron-builder --linux --x64",
     "compile:mac": "./node_modules/.bin/cross-env electron-builder --mac --x64",
     "compile:win": "./node_modules/.bin/electron-builder --win --x64",
-    "prepare-release": "yarn version && echo Successfully incremented version number! Please remember to run git add ., git commit, git push --tags, and git push",
-    "release": "./node_modules/.bin/cross-env CSC_IDENTITY_AUTO_DISCOVERY=false build -mwl"
+    "test": "npm run build-app && jest"
   },
   "devDependencies": {
     "@babel/core": "7.4.5",
     "@babel/plugin-proposal-class-properties": "7.4.4",
     "@babel/plugin-proposal-object-rest-spread": "7.4.4",
     "@babel/preset-env": "7.4.5",
     "@babel/preset-react": "7.0.0",
+    "@jest-runner/electron": "^1.1.1",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
Which dependency needs this?

Reply:
It's `Jest` https://github.com/facebook/jest/issues/6982